packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/e2e/fixture.ts

```diff
@@ -30,66 +30,88 @@ type QuanticGeneratedAnswerE2ESearchFixtures = {
   generatedAnswer: GeneratedAnswerObject;
   search: SearchObject;
   options: Partial<GeneratedAnswerOptions>;
+  withFacets: boolean;
 };
 
 type QuanticGeneratedAnswerE2EInsightFixtures =
   QuanticGeneratedAnswerE2ESearchFixtures & {
     insightSetup: InsightSetupObject;
   };
 
-const exampleQuery = 'exampleQuery';
-
-export const testSearch =
-  quanticBase.extend<QuanticGeneratedAnswerE2ESearchFixtures>({
-    pageUrl,
-    genQaData,
-    options: {},
-    analyticsMode: AnalyticsModeEnum.legacy,
-    search: async ({page}, use) => {
-      await use(new SearchObject(page, searchRequestRegex));
-    },
-    generatedAnswer: async (
-      {page, options, configuration, search, genQaData: data, analytics},
-      use
-    ) => {
-      const generatedAnswerObject = new GeneratedAnswerObject(
-        page,
-        data.streamId,
-        analytics,
-        !!options.answerConfigurationId
-      );
-
-      const streamingUrl = options.answerConfigurationId
-        ? ANSWER_API_URL
-        : `${MACHINE_LEARNING_API_URL}/${data.streamId}`;
-      await generatedAnswerObject.mockStreamResponse(
-        streamingUrl,
-        data.streams,
-        data.streamId
-      );
-
-      await page.goto(pageUrl);
-      await configuration.configure(options);
-      await search.waitForSearchResponse();
-
-      generatedAnswerObject.streamEndAnalyticRequestPromise =
-        generatedAnswerObject.waitForStreamEndAnalytics();
-
-      await search.fillSearchInput(exampleQuery);
-      await search.mockSearchWithGenerativeQuestionAnsweringId(data.streamId);
-      await search.performSearch();
-      await search.waitForSearchResponse();
-
-      await use(generatedAnswerObject);
+export const exampleQuery = 'test';
+
+export const testSearch = quanticBase.extend<QuanticGeneratedAnswerE2ESearchFixtures>({
+  pageUrl,
+  genQaData,
+  options: {},
+  analyticsMode: AnalyticsModeEnum.legacy,
+  withFacets: false,
+  afterEach: async ({page}, use) => {
+    await use();
+    await page.unrouteAll({behavior: 'ignoreErrors'});
+  },
```

Comment:
I'm not super comfortable with this one, 
For me it says that we are doing something wrong in our test and here we are hiding everything under the carpet by unrouting everything and ignoring any errors.

That makes me worried we might be hiding real issues rather than addressing them directly.

Reply:
I agree that we need a proper understanding of why we were getting the error.